src/js/variant-widget/eva-variant-widget-panel.js

```diff
@@ -343,6 +343,333 @@ EvaVariantWidgetPanel.prototype = {
                 case 'zmays_agpv4':
                     defaultRegion = '6:166875000-166876000';
                     break;
+                case 'eva_aaegypti_aaegl2':
```

Comment:
I don't think these should include the `eva_` prefix, based on the others that are already there...

Reply:
Thanks for catching that. 
I've changed it now.